cmd/minishift/cmd/start.go

```diff
@@ -302,13 +317,12 @@ func initSubscriptionManagerFlags() {
 }
 
 // clusterUp downloads and installs the oc binary in order to run 'cluster up'
-func clusterUp(config *cluster.MachineConfig) {
-	oc := cache.Oc{
+func clusterUp(config *cluster.MachineConfig) (*cache.Oc, error) {
```

Comment:
-1, why would _clusterUp_ return _oc_? It would be much better to create _Oc_ outside this method and pass it into _clusterUp_.